Libraries/LibWeb/WebAssembly/Instance.cpp

```diff
@@ -56,6 +60,14 @@ void Instance::initialize(JS::Realm& realm)
 
                 m_exports->define_direct_property(export_.name(), *object, JS::default_attributes);
             },
+            [&](Wasm::GlobalAddress const& address) {
+                Optional<GC::Ptr<Global>> object = cache.get_global_instance(address);
```

Comment:
Note that I'm caching these on the per-realm cache, not on the Instance. From what I could understand, these should be shared b/w instances (or otherwise they wouldn't be 'global')